### ISSUE TITLE
Include DpnII and NlaIII digestion site trimming

### ISSUE THREAD

**Issue by EKingma** [SATAY-LL/Transposonmapper#73]
**Is your feature request related to a problem? Please describe.**
DpnII and NlaIII digestion sites should be trimmed during the processing of the data. These sequences should be trimmed to prevent the contamination of the insertion site sequence with plasmid/other sequences which can affect the accurate mapping during alignment 

**Describe the solution you'd like**
Add the possibility to trim sequences after the DpnII and NlaIII digestion sites before mapping

**Describe alternatives you've considered**

**Additional context**
![image](https://user-images.githubusercontent.com/44368664/155117246-7997ee81-81d5-49b6-9eae-37d40d3d5c77.png)

**Comment by leilaicruz** [comment]
Thanks for the documenting this enhancement to the pipeline! 
For clarification on what you asked for: 

> Add the possibility to trim sequences after the DpnII and NlaIII digestion sites before mapping

Could this be resolved by adding the sequences of the DpnII and NIaIII restriction sites to the adapter.fa file? , which is where we put everything we would like to trim from the data. 

or  also we could ,during the preprocessing (during the part of preparing the dataset for the pipeline ) remove those reads that contain a restriction site , either for DpnII or NIaIII. 

Let me know your thoughts on that

**Comment by leilaicruz** [comment]
Yes, so maybe it is better to remove those reads containing the restriction sites in the pre-processing part and then during the processing in the pipeline just trim the sequencing primer , as we usually do.

**Comment by EKingma** [comment]
Sure, but then it should be mentioned somewhere that this is a step that needs to be done before processing the pipeline

**Comment by leilaicruz** [comment]
yes, exactly :+1:
